aspnetcore/tutorials/razor-pages/validation.md

```diff
@@ -136,7 +162,7 @@ The `DisplayFormat` attribute can be used by itself, but it's generally a good i
 [Range(typeof(DateTime), "1/1/1966", "1/1/2020")]
    ```
 
-It's generally not a good practice to compile hard dates in your models, so using the `Range` attribute and `DateTime` is discouraged.
+It's a best practice to avoid compiling hard dates in models, so using the `[Range]` attribute and `DateTime` is discouraged. Use [Configuration]<xref:fundamentals/configuration/index> for date ranges and other values that are subject to frequent change rather than specifying it in code.
```

Comment:
```suggestion
It's a best practice to avoid compiling hard dates in models, so using the `[Range]` attribute and `DateTime` is discouraged. Use [Configuration](xref:fundamentals/configuration/index) for date ranges and other values that are subject to frequent change rather than specifying it in code.
```